### PR TITLE
[11.0] [MIG] sale_isolated_quotation: Migration to 11.0

### PR DESCRIPTION
**Pull request by rsullivan2704**
Migration of sale-isolated-quotation module from v10 to v11

### CODE REVIEW

File: sale_isolated_quotation/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Usage
+=====
+
+* Create Quotation as normal
+* As user click "Convert to Order", the isolated sales order will be created
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/10.0
```
Comment (nikul-serpentcs) [inline sale_isolated_quotation/README.rst]: Change target `11.0`

File: sale_isolated_quotation/data/ir_sequence_data.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline sale_isolated_quotation/data/ir_sequence_data.xml]: @rsullivan2704 IMO here direct use `<odoo noupdate="1">` and remove `data`

File: sale_isolated_quotation/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# © 2017 Ecosoft (ecosoft.co.th).
```
Comment (nikul-serpentcs) [inline sale_isolated_quotation/models/__init__.py]: Remove Copyright from all `__init__` file

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by rafaelbn** [comment]
Hi @rsullivan2704 could you please attend comments from the review of @nikul-serpentcs ? Thanks

File: sale_isolated_quotation/README.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Kitti U. <kittiu@ecosoft.co.th>
```
Comment (kittiu) [inline sale_isolated_quotation/README.rst:49]: @rsullivan2704 You can add yourself as contributor

**Review by kittiu — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rafaelbn** [comment]
Hi @rsullivan2704 we cannot merge this with travis red, please @mreficent could help here?

File: sale_isolated_quotation/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# © 2017 Ecosoft (ecosoft.co.th).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Sales - Isolated Quotation',
+    'version': '11.0.1.0.0',
+    'author': 'Ecosoft,Odoo Community Association (OCA)',
+    'category': 'Sales',
+    'website': 'https://github.com/OCA/sale-workflow/tree/11.0/sale_isolated_quotation',
```
Comment (MiquelRForgeFlow) [inline sale_isolated_quotation/__manifest__.py]: line too long

better option:
```suggestion
    'website': 'https://github.com/OCA/sale-workflow',
```

**Comment by rafaelbn** [comment]
@rsullivan2704 please fix it, it's just a line :smile: 

Thanks @mreficent

**Review by rafaelbn — APPROVED** [review]
@rsullivan2704 I approve this but this cannot be merge until you change that line for travis green

**Pull request merged**